.gitlab-ci.yml

```diff
@@ -119,6 +119,8 @@ build-in-docker:
     CMAKE_BUILD_TYPE: "Release"
     EXTRA_CMAKE_FLAGS: ""
   script:
+    # To be sure that AutoGenerated is correct, remove it first and check later that it's unchanged.
+    - rm -rf framework/tlvf/AutoGenerated common/beerocks/tlvf/AutoGenerated
```

Comment:
Shall we also remove `./common/beerocks/tlvf/AutoGenerated` ?

Reply:
It's already there:
```
    - rm -rf framework/tlvf/AutoGenerated common/beerocks/tlvf/AutoGenerated
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^
```